packages/manager/apps/hub/src/index.less

```diff
@@ -2,7 +2,9 @@
 @import '@ovh-ux/manager-account-sidebar/src/variables.less';
 
 .hub-main-view {
-  margin-right: @ovh-sidebar-width;
+  &_sidebar_expanded {
```

Comment:
Nested selector isn't needed I think.

This should work: `.hub-main-view_sidebar_expanded`